extensions/mongodb/src/test/java/com/hazelcast/jet/mongodb/MongoSourceTest.java

```diff
@@ -298,7 +298,8 @@ public void testStream_whenWatchAll() {
         String connectionString = mongoContainer.getConnectionString();
 
         Stream<?> builder = MongoSourceBuilder.stream(() -> MongoClients.create(connectionString));
-        builder = streamFilters(builder);
+        builder = streamFilters(builder)
+                .checkResourceExistence(ResourceChecks.ONLY_INITIAL);
```

Comment:
`ONLY_INITIAL` is meant for SQL only according to javadoc

Reply:
7831956e3b2ce0787afc566a54b89fa469b724b0